aesara/sparse/basic.py

```diff
@@ -1133,7 +1133,8 @@ def grad(self, inputs, gout):
         (x,) = inputs
         (gz,) = gout
         gx = dense_from_sparse(gz)
-        gx = at.patternbroadcast(gx, x.broadcastable)
```

Comment:
Is there a preference for `var.broadcastable` vs `var.type.broadcastable`?

Reply:
`var.type` is more explicit, and less likely to confuse Mypy.